server/routes/healthy.js

```diff
@@ -0,0 +1,37 @@
+const databaseService = require('../services/database-service')
+const messageService = require('../services/message-service')
+
+module.exports = {
+  method: 'GET',
+  path: '/healthy',
+  options: {
+    handler: async (request, h) => {
+      const calculationSender = messageService.getCalculationSender()
+      const scheduleSender = messageService.getScheduleSender()
+
+      const unavailableServices = []
+
+      if (!await databaseService.isConnected()) {
+        unavailableServices.push('database')
+      }
+
+      if (!calculationSender.isConnected()) {
+        unavailableServices.push('calculation queue')
+      }
+
+      if (!scheduleSender.isConnected()) {
+        unavailableServices.push('schedule queue')
+      }
+
+      let message = 'ok'
+      let statusCode = 200
+
```

Comment:
down to personal preference, but I'd avoid using `let` wherever possible to keep values immutable, as having default values that are overriden later on in the code can be a source of bugs.

Reply:
Fair point, I do try to avoid `let` as well but I also try to avoid having multiple return statements in a function as that can cause trouble too. I've switched to using short ternary statements to avoid the need for `let`. This is clearer and probably less error-prone although can mislead test coverage reports if only one side of the conditional is tested (both cases are currently tested).

At the same time, I've put a bit more thought into the logic for determining which services are disconnected and refactored it into a shorter format.

Reply:
Some nice succinct changes there!